src/views/v2/commerce/gateway/paypal/advanced-payments.php

```diff
@@ -25,7 +25,7 @@
 ?>
 
 <!-- Advanced credit and debit card payments form -->
-<div class="tribe-tickets__commerce-checkout-paypal-advanced-payments-container">
+<div class="tribe-tickets__commerce-checkout-paypal-advanced-payments-container tribe-common-a11y-hidden">
```

Comment:
We need to add the `@since TBD` and update the `@version TBD` for templates whenever we make changes.